docs/environment_variables.md

```diff
@@ -40,7 +40,7 @@ These are the environment variables and values that aid in troubleshooting the C
 | Variable | Value | Description |
 |-|-|-|
 MicrosoftInstrumentationEngine_DebugWait|1|Suspends the process until the debugger is attached.
-MicrosoftInstrumentationEngine_FileLogPath|"[FULL PATH TO LOGGING FILE]"|File to host the event logs. This requires LogLevel to be set.
+MicrosoftInstrumentationEngine_FileLogPath|"[FULL PATH TO LOGGING FILE]"|File to host the event logs. This requires LogLevel to be set. If multiple processes are profiled, then use a directory (end with slash) so that each process generates its own log file `ProfilerLog_[processId].txt`
```

Comment:
I believe that `<path>\*` is also supported, which is probably safer from getting the ending slash from being accidentally forgotten or parsed off.

Reply:
Good catch, double checked the code and it allows `<path>\.` so I'll update this comment accordingly